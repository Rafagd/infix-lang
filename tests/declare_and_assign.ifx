integer is i32;
integer = 1337;
void print integer;

float is f32;
float = 0.1337;
void print float
