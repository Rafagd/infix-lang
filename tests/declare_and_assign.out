1337
0.133700
